homeassistant/components/light/tellstick.py

```diff
@@ -5,126 +5,84 @@
 https://home-assistant.io/components/light.tellstick/
 """
 from homeassistant.components.light import ATTR_BRIGHTNESS, Light
-from homeassistant.const import EVENT_HOMEASSISTANT_STOP
-
+from homeassistant.components.tellstick import (TellstickDevice,
+                                                TELLCORE_REGISTRY,
+                                                DISCOVER_DEVICES,
+                                                DISCOVER_CONFIG)
 REQUIREMENTS = ['tellcore-py==1.1.2']
```

Comment:
Tellcore is needed here as well because I'm importing constants from the tellcore module. 
The light/switch platforms do have a dependency to the main component, so should I just remove the REQUIREMENTS from here completely.